Prepare

```diff
@@ -5,5 +5,5 @@ set -e
 
 sh ./autogen.sh
 
-CFLAGS="-O0 -g -Wall" ./configure --with-pic --enable-maintainer-mode --enable-dependency-tracking --enable-shared=yes
+CFLAGS="-O0 -g -Wall" ./configure --with-pic --enable-dependency-tracking --enable-shared=yes
```

Comment:
and if we add `LT_INIT([pic-only])` to `configure.ac` then this should should also eliminate the need for `--with-pic`
This can still be overwritten with --without-pic